tools/ci.psm1

```diff
@@ -593,3 +594,212 @@ function Invoke-AppveyorFinish
+                # This allows us to prevent regressions to a critical engine experimental feature.
+                $sudoPesterParam.Remove('Path')
+            } else {
+                # If a non-empty string or array is specified for the feature name, we only run those test files.
+                $sudoPesterParam['Path'] = $testFiles
+            }
+            $passThruResult = Start-PSPester @sudoPesterParam -Title "Pester Experimental Sudo - $featureName"
+            $sudoResultsWithExpFeatures += $passThruResult
+        }
+
+        # Determine whether the build passed
+        try {
+            $allTestResultsWithNoExpFeature = @($pesterPassThruNoSudoObject, $pesterPassThruSudoObject)
+            $allTestResultsWithExpFeatures = $noSudoResultsWithExpFeatures + $sudoResultsWithExpFeatures
+            # this throws if there was an error
+            $allTestResultsWithNoExpFeature | ForEach-Object { Test-PSPesterResults -ResultObject $_ }
+            $allTestResultsWithExpFeatures  | ForEach-Object { Test-PSPesterResults -ResultObject $_ -CanHaveNoResult }
+            $result = "PASS"
+        }
+        catch {
+            $resultError = $_
+            $result = "FAIL"
+        }
+
+        try {
+            $ParallelXUnitTestResultsFile = "$pwd/ParallelXUnitTestResults.xml"
+            Start-PSxUnit -ParallelTestResultsFile $ParallelXUnitTestResultsFile
+            # If there are failures, Test-XUnitTestResults throws
+            Test-XUnitTestResults -TestResultsFile $ParallelXUnitTestResultsFile
+        }
+        catch {
+            $result = "FAIL"
+            if (!$resultError)
+            {
+                $resultError = $_
+            }
+        }
+
+        if ($createPackages) {
+
+            $packageParams = @{}
+            $packageParams += @{ReleaseTag=$releaseTag}
+
+            # Only build packages for branches, not pull requests
+            $packages = @(Start-PSPackage @packageParams -SkipReleaseChecks)
+            foreach($package in $packages)
+            {
+                # Publish the packages to the nuget feed if:
+                # 1 - It's a Daily build (already checked, for not a PR)
+                # 2 - We have the info to publish (NUGET_KEY and NUGET_URL)
+                # 3 - it's a nupkg file
+                if($isDailyBuild -and $NugetKey -and $env:NUGET_URL -and [system.io.path]::GetExtension($package) -ieq '.nupkg')
+                {
+                    Write-Log "pushing $package to $env:NUGET_URL"
+                    Start-NativeExecution -sb {dotnet nuget push $package --api-key $NugetKey --source "$env:NUGET_URL/api/v2/package"} -IgnoreExitcode
+                }
+            }
+            if ($IsLinux)
+            {
+                # Create and package Raspbian .tgz
+                Start-PSBuild -PSModuleRestore -Clean -Runtime linux-arm -Configuration 'Release'
+                Start-PSPackage @packageParams -Type tar-arm -SkipReleaseChecks
+            }
```

Comment:
```suggestion
            }
            
            if ($isDailyBuild)
            {
                New-TestPackage -Destination "${env:SYSTEM_ARTIFACTSDIRECTORY}"
            }
```

Reply:
maybe you moved this somewhere else?

Reply:
Not sure, I know I lost a few of the daily build checks in the move because my git likes to act up